admin_manual/maintenance/moving-data-folders.rst

```diff
@@ -0,0 +1,75 @@
+============================
+How To Move a Data Directory
+============================
+
+If you need to move your ownCloud data directory from its current location to
+somewhere else, here is a manual process that you can take to make it happen.
+
+.. NOTE:: 
+   This example assumes that:
+
+   - The current folder is: `var/www/owncloud/data`
+   - The new folder is: `/mnt/owncloud`
+   - You’re using Apache as your webserver
+
+1. Stop Apache
+2. Use rsync to sync the files from the current folder to the new one the folders -> 
+3. Create a symbolic link from the new directory to the old one 
+4. Double-check `the directory permissions`_ on the new directory 
+5. Restart Apache
+
+To save time, here’s the commands which you can copy and use::
+
+  apachectl -k stop 
+  rsync -avz var/www/owncloud/data /mnt/owncloud
+  ln -s /mnt/owncloud /www/owncloud/data
+  apachectl -k graceful 
+
+.. NOTE:: 
+   If you’re on Centos, try `service httpd stop` and `service httpd start`.
+   If you’re on Debian/Ubuntu try `sudo service apache2 stop` and `sudo service apache2 start`
+
+Updating The Database Manually
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to manually change the location of the data folder in the database,
+run the SQL below:
+
+.. code-block::
+   
+  UPDATE oc_storages SET id='local::/var/www/owncloud/data' WHERE id='local::/mnt/owncloud'
+
+Hardcoded Database Path Variables 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+One thing worth noting is that individual apps may reference the data directory
+separate from the core system configuration. If so, then you will need to find
+which applications do this, and change them as needed. 
+
+For example, if you listed the application configuration by running `occ
+config:list`, then you might see output similar to that below:
```

Comment:
Github weirdly formatted this block. I assume it's a Github bug and you didn't get that in the final version